### PR TITLE
Reader: Only link to Discover for en users

### PR DESCRIPTION
**Pull request by mjangda**
Hides Discover links for non-`en` users since the content is currently only in English.

## To Test

- Set interface language to English from Me > Account Settings.
- Go to Reader.
- Make sure "Discover" shows in sidebar.

- Switch language to something not English.
- Go to Reader.
- Make sure "Discover" is no longer visible in sidebar

### CODE REVIEW

File: client/reader/discover/helper.jsx [hunk 1]
```diff
@@ -5,7 +5,18 @@ var find = require( 'lodash/collection/find' ),
 	get = require( 'lodash/object/get' ),
 	url = require( 'url' );
 
+/**
+ * Internal Dependencies
+ */
+var config = require( 'config' ),
```
Comment (gwwar) [inline client/reader/discover/helper.jsx]: const?

File: client/reader/discover/helper.jsx [hunk 2]
```diff
@@ -5,7 +5,18 @@ var find = require( 'lodash/collection/find' ),
...
+	userUtils = require( 'lib/user/utils' );
+
 module.exports = {
+	isEnabled: function() {
+		return config.isEnabled( 'reader/discover' ) &&
+			userUtils.getLocaleSlug() === 'en';
```
Comment (gwwar) [inline client/reader/discover/helper.jsx]: Is this guaranteed to not include the country code?

**Comment by gwwar** [comment]
:+1: Looks good.

In a non-english locale, http://calypso.localhost:3000/discover will redirect to discover.wordpress.com. Is this desired?

**Comment by mjangda** [comment]
> In a non-english locale, http://calypso.localhost:3000/discover will redirect to discover.wordpress.com. Is this desired?

The redirect should only happen for logged out users. We're also not blocking access to `/discover` for non-`en` users; just hiding it.

**Pull request merged**